components/modal/index.css

```diff
@@ -22,6 +22,13 @@ governing permissions and limitations under the License.
 	--spectrum-dialog-fullscreen-margin: 32px;
 	--spectrum-dialog-max-height: 90vh;
 	--spectrum-dialog-max-width: 90%;
+
+	--spectrum-dialog-background-color: var(--spectrum-gray-100);
+	--spectrum-dialog-confirm-border-radius: var(--spectrum-corner-radius-100);
+	--spectrum-dialog-confirm-exit-animation-duration: 130ms;
+	--spectrum-dialog-confirm-entry-animation-duration: 250ms;
+	--spectrum-dialog-confirm-entry-animation-delay: 160ms;
+	--spectrum-dialog-transition-animation-duration: 130ms;
```

Comment:
Since this is Modal and not Dialog, it would likely be best to rename these to use `--spectrum-modal` instead of `--spectrum-dialog`. @pfulton would we need to mark this as a breaking change if we were to do something like that?